### PR TITLE
Order amendment details alphabetically

### LINKED ISSUE #2474: "Details of Known Amendments" should be in alphabetical order
On https://xrpl.org/resources/known-amendments the list of amendment details was originally in alphabetical order but some of the newer ones have been added to the top instead. We should move the detailed sections so that they're back in alphabetical order.

(The entries in the table are in chronological order and should remain that way.)

### PR DESCRIPTION
**Pull request by amarantha-k**
This PR closes #2474

### CODE REVIEW

File: resources/known-amendments.md [hunk 1]
```diff
@@ -360,6 +223,28 @@ This amendment was intended to add support for several types of crypto-condition
 However, the amendment was added to `rippled` v0.60.0 before implementation was complete. As a result, this amendment ID refers to incomplete code which does almost nothing. Modifying the existing amendment to add support for other crypto-conditions would cause a conflict with old versions of the amendment already in released software. If a future release adds support for additional crypto-conditions, it must use a new and different amendment ID.
 
 
+### DID
```
Comment (DennisDawson) [inline resources/known-amendments.md:226]: Decentralized Identifier comes before DeletableAccounts, but DID doesn't. Should the heading be Decentralized Identifier?

File: resources/known-amendments.md [hunk 2]
```diff
@@ -725,6 +610,20 @@ Without this amendment, it is possible to have a [payment path](../docs/concepts
 With this amendment, those payments fail with the [`temBAD_PATH_LOOP` result code](../docs/references/protocol/transactions/transaction-results/tem-codes.md) instead.
 
 
+### fixAMMOverflowOffer
```
Comment (DennisDawson) [inline resources/known-amendments.md:613]: Should fix AM_M_OverflowOffer come after fixAm_e_ndmentMajorityCalc?

File: resources/known-amendments.md [hunk 3]
```diff
@@ -1379,13 +1349,33 @@ With this amendment enabled, a `TrustSet` transaction with [`tfSetfAuth` enabled
...
-|:-------------|:-------------|
+|:-------------|:-----------------|
 | Amendment ID | C98D98EE9616ACD36E81FDEB8D41D349BF5F1B41DD64A0ABC1FE9AA5EA267E9C |
-| Status       | In Development |
+| Status       | Open for Voting |
 | Default Vote (Latest stable release) | No |
 | Pre-amendment functionality retired? | No |
 
-Adds "cross-chain bridges" for synchronizing assets between different networks, such as Mainnet and a sidechain. Standards draft: [XLS-38d Cross-Chain Bridge](https://github.com/XRPLF/XRPL-Standards/blob/master/XLS-38d-XChainBridge/README.md).
+Adds cross-chain bridges, enabling the transfer of digital assets between networks, such as Mainnet and a sidechain.
```
Comment (DennisDawson) [inline resources/known-amendments.md]: ...between networks (for example, between Mainnet and a sidechain).

**Review by DennisDawson — APPROVED** [review]
A few suggestions, but good to go.

File: resources/known-amendments.md (hunk 1 above)
Comment (amarantha-k) [reply resources/known-amendments.md:226]: No, the heading is the name of the amendment, hence will leave it as DID, or AMM and so on.

File: resources/known-amendments.md (hunk 2 above)
Comment (amarantha-k) [reply resources/known-amendments.md:613]: No, uppercase letters come before lowercase.

**Pull request merged**